WeakAuras/Prototypes.lua

```diff
@@ -7955,6 +7955,8 @@ Private.event_prototypes = {
       trigger.unit = trigger.unit or "target";
       local ret = [[
         unit = string.lower(unit)
+        local name, realm = WeakAuras.UnitNameWithRealm(unit)
+        local smart = %s
```

Comment:
You can simplfy this, smart is always false since there's no "group" unit (nor will there ever be one)